homeassistant/components/easyenergy/strings.json

```
@@ -60,12 +60,12 @@
       "description": "Requests gas prices from easyEnergy.",
       "fields": {
         "config_entry": {
-          "name": "Config Entry",
+          "name": "Config entry",
           "description": "The configuration entry to use for this action."
         },
         "incl_vat": {
-          "name": "VAT Included",
-          "description": "Include or exclude VAT in the prices, default is true."
+          "name": "VAT included",
+          "description": "Whether the prices should be retrieved including value added tax."
```

```suggestion
          "description": "Whether the prices should include VAT."
```
I think we should keep VAT as it is a well known term, and it makes translations also keep it

---

In German the abbreviation is "MwSt." so nobody says that or spells that out and the description uses "Mehrwertsteuer" instead. ;-)

But we can use "VAT" in both name and description in English.

---

English is our main language and VAT is the correct abbreviation for this 😉

---

No problem, I just try to add some more detail to descriptions where possible.
But I understand that many people don't even know what words VAT stands for 😃 

Already changed as suggested.